rules/default.json

```diff
@@ -38,9 +38,9 @@
         },
         {
             "name": "path-keys-no-trailing-slash",
-            "object": "pathInfo<--This doesnt exist so this whole rule is broke",
-            "enabled": false,
-            "description": "path item keys should not end with a slash",
+            "object": "openapi",
```

Comment:
I took this from [oas-linter/rules.yaml](https://github.com/Mermade/oas-kit/blob/master/packages/oas-linter/rules.yaml) but im not sure its right. These aren't keys on `openapi` they're multiple keys on `paths`. Going to have to defer to @MikeRalphson on this one.

Reply:
Yes, this was broken in multiple ways. :disappointed: The fixes are here: https://github.com/Mermade/oas-kit/commit/f17105c66e2dbd48e9c0e0d46c263cf65d7ce506

Reply:
You'll need to bump your dep on `oas-validator` to `^1.1.7`